DataFormats/Detectors/TRD/include/DataFormatsTRD/Tracklet64.h

```diff
@@ -26,6 +26,8 @@
 //
 ////////////////////////////////////////////////////////////////////////////
 
+#include "GPUCommonDef.h"
+#include "GPUCommonRtypes.h" // OS: FIXME: is this needed?
```

Comment:
Please don't include both, just include the GPUCommonRTypes.h if you need ClassDef in a header that is supposed to be compiled on GPU.

Reply:
Ah, you mean not both GPUCommonRtypes.h and Rtypes.h, right? Because GPUCommonDef.h is needed. This is changed now